gdg_events/google_io_extended.md

```diff
@@ -2,6 +2,16 @@
 
 ![](https://events.google.com/io2015/images/io15-color.png)
 
+Every year, Google hosts developers around world in a software developer-focused conference held in San Francisco, California to showcase innovations in areas like Android mobile development, Modern front-end tools, infrastructure & DevOps, wearables, phones etc. 
+
+Google I/O features highly technical, in-depth sessions focused on building web, mobile, and enterprise applications with Google and open web technologies such as Android, Chrome, Chrome OS, Google APIs, Google Web Toolkit, App Engine, and more.
+
+As interesting as it sounds, many developers around the world feel left out as they are unable to attend the Google I/O event every year. Students and Industry Experts are always vying to get a glimpse of this much awaited event, but in vain.
+
+To overcome this, Google has created an initiative which helps extending this conference through various communities like the Google Developer Groups, Google Business Groups etc, titled as `Google I/O Extended` to different locations through interested organizers for the benefit of developers, students, start-up junkies around the world.
```

Comment:
A summary of the [organiser guide](https://docs.google.com/presentation/d/1rUwLzz3-Hgcs9eu1uph4En1FUCcpTQspw-xpK4Kaoms/pub) and then a link to it would suffice I think.